### PR TITLE
Add encryption_configuration parameter to BigQuery operators

### PR DESCRIPTION
**Pull request by VladaZakharova**
This PR adding encryption_configuration parameter to BigQueryValueCheckOperator, BigQueryColumnCheckOperator, BigQueryGetDataOperator and BigQueryIntervalCheckOperator to add CMEK key support for these operator.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/google/cloud/operators/bigquery.py [hunk 1]
```diff
@@ -336,6 +343,13 @@ class BigQueryValueCheckOperator(_BigQueryDbHookMixin, SQLValueCheckOperator):
     :param sql: SQL to execute.
     :param use_legacy_sql: Whether to use legacy SQL (true)
         or standard SQL (false).
+    :param encryption_configuration: [Optional] Custom encryption configuration (e.g., Cloud KMS keys).
```
Comment (Lee-W) [inline airflow/providers/google/cloud/operators/bigquery.py]:
```suggestion
    :param encryption_configuration: (Optional) Custom encryption configuration (e.g., Cloud KMS keys).
```

nitpick: let's unify the docstring convention in this module.

**Review by Lee-W — APPROVED** [review]

**Review by shahar1 — APPROVED** [review]

File: airflow/providers/google/cloud/operators/bigquery.py (hunk 1 above)
Comment (Lee-W) [reply airflow/providers/google/cloud/operators/bigquery.py]: It seems we've not yet unified this in this module. I'll approve this PR, but I would like to wait for this nitpick to be addressed. Thanks!

**Review by Lee-W — APPROVED** [review]

File: airflow/providers/google/cloud/operators/bigquery.py (hunk 1 above)
Comment (VladaZakharova) [reply airflow/providers/google/cloud/operators/bigquery.py]: Didn't notice other places 🥲 
Thank you!
I have changed it in every place in this file

Comment (Lee-W) [reply airflow/providers/google/cloud/operators/bigquery.py]: No worries 🙂  Thank you so much for the prompt update. I'll merge it once CI pass 🙌

**Pull request merged**